### PR TITLE
0.7: Updated the release notes

### PR DESCRIPTION
**Pull request by dobrienCP**

### CODE REVIEW

File: docs_outdoornav_user_manual/release_notes.mdx [hunk 1]
```diff
@@ -6,6 +6,36 @@ toc_min_heading_level: 2
...
+## 0.7.0
+
+### New Features
+
+-   Goal terminology has been removed from the mission generation nomenclature (see 
+    [Definitions](/docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode#definitions)) 
+    Users can now add tasks, apply final headings, and set navigation tolerances 
+    to any Waypoint in a mission.
+-   Drag and Drop of waypoints now available in Edit Mode.
+-   New Waypoints can be inserted between existing Waypoints in a mission.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/release_notes.mdx]: Should waypoints and mission be capitalized in all cases?

File: docs_outdoornav_user_manual/release_notes.mdx [hunk 2]
```diff
@@ -6,6 +6,36 @@ toc_min_heading_level: 2
...
+-   Drag and Drop of waypoints now available in Edit Mode.
+-   New Waypoints can be inserted between existing Waypoints in a mission. 
+-   Mission API now available to create/edit/load missions, waypoints and tasks.
+-   Mission execution via mission ID is now available.
+-   The base station location is now displayed in the UI after carrying out an automated survey.
+-   New coloring scheme for GNSS status icons to provide more accurate information.
+
+### Bug Fixes
+
+-   1069: Fixed the test script for versions pre-0.5
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/release_notes.mdx]: What does the "pre-0.5" part mean? How does that apply to 0.7?

File: docs_outdoornav_user_manual/release_notes.mdx [hunk 3]
```diff
@@ -6,6 +6,36 @@ toc_min_heading_level: 2
...
+-   996: Axis camera missing ptz_state
+
+### Known Issues
+
+-   131: Software upgrade process not documented fully
+-   609: Realsense D435 collision avoidance not fully tuned
+-   751: Wireless icon in UI doesn't work for Ubiquity hardware
+-   1134: Display/Hide Datum point not working
+-   1137: Refreshing page re-enabled edit button while mission running
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/release_notes.mdx]: "re-enabled" -> "re-enables"

File: docs_outdoornav_user_manual/release_notes.mdx [hunk 4]
```diff
@@ -6,6 +6,36 @@ toc_min_heading_level: 2
...
+-   1138: Issues with Greying out waypoints in edge cases
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/release_notes.mdx]: "Greying" -> "greying"
waypoints -> Waypoints

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 5]
```diff
@@ -92,6 +92,14 @@ over the planned route as can be seen in the following figure.
   </figure>
 </center>
 
+:::note
+
+If the UGV is manually driven outside of the constrained replanning area 
+while a mission is running, the mission will need to be restarted as the UGV
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: Capitalize Mission in both cases?

**Review by jhiggins-cpr — CHANGES_REQUESTED** [review]
Can you also update navigation_packages.mdx as part of releasing the 0.7.0 docs so that "Latest" points to 0.7.0, which will now be part of the same documentation package, rather than an external link to the sphinx-based docs?

File: docs_outdoornav_user_manual/release_notes.mdx (hunk 2 above)
Comment (dobrienCP) [reply docs_outdoornav_user_manual/release_notes.mdx]: Ah I just copied the title from the ticket. I'll change the "for versions pre-0.5" and just say something like "Fixed the test script for older versions"

Comment (jhiggins-cpr) [reply docs_outdoornav_user_manual/release_notes.mdx]: I might just skip this bug fix. I'm unclear on how fixes to the test script for 0.4 installations are part of the 0.7 release. Customers might be confused also.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**